distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Pattern.enso

```diff
@@ -69,7 +69,7 @@ type Pattern
 
        This method will _always_ return a vector. If no splits take place, the
        vector will contain a single element.
-    split : Text -> (Matching_Mode.First | Integer | Regex_Mode) -> Vector Text
```

Comment:
@radeusgd This was the only case I found that had First but not Last.

Reply:
Okay, thanks for an exhaustive check. I suspect we have updated the implementation but forgot to update the 'interface' and since the latter does not really have a code connection to the former, its more of a documentation, we did not notice.